src/main/java/io/dropwizard/actors/RabbitmqActorBundle.java

```diff
@@ -34,4 +35,8 @@ public void initialize(Bootstrap<?> bootstrap) {
     }
 
     protected abstract RMQConfig getConfig(T t);
+
+    protected abstract MetricRegistry getMetricRegistry(T t);
```

Comment:
This is to provide user with capability to supply custom MetricRegistry if required. Also this would provide user option to enable/disable instrumentation completely.